app/scripts/controllers/reviewRegistration.js

```diff
@@ -280,4 +329,117 @@ angular.module('confRegistrationWebApp')
     $scope.hasPendingCheckPayment = function(payments){
       return _.some(payments, { paymentType: 'CHECK', status: 'PENDING' });
     };
+
+    ////// START EVENT-433 //////
```

Comment:
@OzzieOrca I moved all of the new logic into new services: `payment`, `registration`, and `error`. It is now ready for another code review.